index.js

```diff
@@ -77,6 +77,12 @@ Prober.prototype.notok = function notok() {
     }
 };
 
+Prober.prototype.setEnabled = function setEnabled(enabled) {
+    if (typeof enabled === 'boolean') {
```

Comment:
I feel it's better to do assert at the source, for instance:

If (flipr.get('enableAirlock') === 'true') {
   Prober.enabled()
}

instead of potentially error prone code like:
Prober.setEnabled(flipr.get('enableAirlock'));

Thoughts?